package.json

```diff
@@ -50,7 +50,9 @@
     "mocha": "^3.0.2",
     "taskcluster-lib-testing": "^1.0.0"
   },
+  "engine-strict": true,
   "engines": {
-    "node": "5"
+    "node": "6",
```

Comment:
If this is going to be 6, let's do 6.10.0

I think 7.7.1 will work both here and travis though!